cmd/version/version.go

```diff
@@ -78,8 +84,64 @@ func NewCmdVersion(rootCmd *cobra.Command) *cobra.Command {
 			}
 
 			util.TablePrinter(util.VersionColumnDefinition, rows, printers.PrintOptions{Wide: true})
+			getCliLatestVersion(rootCmd.Version)
 		},
 	}
 	cmd.SetUsageTemplate(versionCmdHelp)
 	return cmd
 }
+
+func getCliLatestVersion(currVersion string) {
+	// getting the latest version of openebsctl from sigs.k8s.io/krew-index
+	resp, err := http.Get("https://raw.githubusercontent.com/kubernetes-sigs/krew-index/master/plugins/openebs.yaml")
+	util.CheckErrDefault(err, "Error reading response body")
```

Comment:
```suggestion
      if err != nil {
         return something
    }
```
you might not need to pass the error around for checking the error, 

Don't commit it from here, do it from local machine

Reply:
Okay, deleted it :smile: